docs/examples/parameter_recovery_in_logspace_demo.md

```diff
@@ -0,0 +1,204 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernel_info:
+  name: datana
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Parameter Recovery in Log-Space Demo
+
+In this demo, we show a parameter recovery using `psignifit`, in the case where we want to fit a sigmoid in log space. 
+
+This demo is very similar to the "Parameter Recovery Demo", and you should have a look at that one first, but it also shows how to fit the sigmoid in log-space, and how to recover width, threshold, and confidence intercals in the original simulus space.
+
+We will cover the following steps:
+
+  1. Simulating data with known parameters.
+  
+  2. Fitting the model to the simulated data in logspace.
+
+  3. Converting the fitted parameters back to stimulus space.
+  
+  4. Comparing the recovered parameters to the original parameters.
+
+
+```{code-cell} ipython3
+import numpy as np
+from matplotlib import pyplot as plt
+
+import psignifit
+from psignifit import psigniplot
+from psignifit.sigmoids import sigmoid_by_name
+```
+
+First, we set parameters using which the data will be simulated
+
+```{code-cell} ipython3
+stim_range = [1.0, 1000.0]
+threshold = 134
+
+lambda_ = 0.0232
+gamma = 0.1
+nsteps = 20
+num_trials = 50000
+# We are going to fit a log-Weibull sigmoid
+sigmoid_name = 'weibull'
+sigmoid = sigmoid_by_name('weibull')
+# We choose levels spaced logarithmically in the stimuli space, because that is often what happens
+# in experiments where the fit is done in logarithmic space
+stimulus_level = np.logspace(np.log10(stim_range[0]), np.log10(stim_range[1]), nsteps, base=10)
+# However, one could just as well space the stimuli linearly
+#stimulus_level = np.linspace(stim_range[0], stim_range[1], nsteps)
+```
+
+Using the sigmoid object, we can simulate percent correct values for each stimulus level, without introducing 
+additional variability.
+
+We generate the data on the data and parameters transformed in log-space, since the fit it's going to be made there.
+
+```{code-cell} ipython3
+logspace_stimulus_level = np.log(stimulus_level)
+log_threshold = np.log(threshold)
+log_width = 2.3  # Width in log-space
+
+perccorr = sigmoid(logspace_stimulus_level, log_threshold, log_width, gamma, lambda_)
+```
+
+This is what the generated data and threshold look like in the stimulus space
+
+```{code-cell} ipython3
+fig, ax = plt.subplots()
+ax.semilogx(stimulus_level, perccorr, marker='o')
+ax.set_xlabel("Stimulus Level")
+ax.set_ylabel("Percent Correct")
+plt.axvline(threshold, color='r')
+pc = (1 - gamma - lambda_) / 2 + gamma
+plt.axhline(pc, color='r')
```

Comment:
need a `;` at the end of the line, or the page is going to show a Line2D object

<img width="528" alt="image" src="https://github.com/user-attachments/assets/a8149439-7cfa-413f-b608-e983591f83dc" />